### PR TITLE
fix(ext/node): add symmetric keygen

### PR DESCRIPTION
**Pull request by littledivy**
Towards #18455

### CODE REVIEW

File: cli/tests/node_compat/test/parallel/test-crypto-secret-keygen.js
Comment (bartlomieju) [inline cli/tests/node_compat/test/parallel/test-crypto-secret-keygen.js:1]: Great!

**Review by bartlomieju — APPROVED** [review]
LGTM! Thanks Divy

**Comment by ry** [comment]
Should we call this a fix instead of a feat? It's really just fixing the compatibility layer and I think can go out in a patch release.

**Comment by bartlomieju** [comment]
> Should we call this a fix instead of a feat? It's really just fixing the compatibility layer and I think can go out in a patch release.

Good point. Though in the last release we just merged `feat` PRs from `ext/node`.

**Pull request merged**